packages/pwa-kit-dev/src/utils/resolver-utils.test.js

```diff
@@ -140,12 +146,33 @@ describe('resolverUtils', () => {
 
                 extensions: ['module-extension-a', 'module-extension-b', 'module-extension-c'],
                 expected: [
-                    path.join(process.cwd(), 'app', 'routes'),
+                    path.join(process.cwd(), 'app', 'overrides', 'app', 'routes'),
```

Comment:
It looks like we're not checking locally in "app/routes", but we do check the local _overrides_ folder 🤔 I'm curious why that is.